examples/ColumnGroupsExample.js

```diff
@@ -65,6 +65,13 @@ class ColumnGroupsExample extends React.Component {
             width={150}
           />
         </ColumnGroup>
+        <Column
```

Comment:
Yes, that is uninteded behaviour. It is an issue with the generation of column header from the data structure. Before my changes it was not possible to use Columns next to ColumnGroups and since ColumnGroups may have two header levels (should be more in the future) the `render` method of `FIxedDataTable.js` adds a row for the column headers and - if there are ColumnGroups - one for the ColumnGroup header.
As you outlined [here](https://github.com/schrodinger/fixed-data-table-2/issues/174#issuecomment-311676994) that row spanning is not possible at the moment and not the focus of this feature branch. Just added the demo to show that is generally possible - but currently with the downside of a duplicated column header.
Do you have any better ideas?
My basic idea was to implement a new Column(Group) header rendering in another PR after this PR became merged.